jupyterhub/templates/singleuser/secret.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.singleuser.extraFiles }}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: {{ include "jupyterhub.singleuser.fullname" . }}
+  labels:
+    {{- include "jupyterhub.labels" . | nindent 4 }}
+type: Opaque
+{{- with include "jupyterhub.extraFiles.data" .Values.singleuser.extraFiles }}
```

Comment:
what's the max size for a secret? Is there a reasonable risk of exceeding the max secret size with this by putting all mounted files in one secret?

Reply:
Yes, that would be worth a mention!

I'm not sure all k8s clusters will have the same limitations which relates to the k8s Secret storage which probably is etcd generally, but 3 MB is the upper bound I've experienced on GKE.

Also, base64 encoding leads to an increase of 33%. This is probably only a limitation worth mentioning for binaryData.

## Action point
- [ ] Create a note about the file size limitations (hub / singleuser get one k8s secret to store the files in base64 encoding, and they can maximally be ~3 MB in total)

Reply:
I've added a docs about this under a Limitation section.